src/components/Layout.js

```diff
@@ -3,7 +3,7 @@ import { Helmet } from 'react-helmet'
 import Footer from '../components/Footer'
 import useSiteMetadata from './SiteMetadata'
 import { withPrefix } from 'gatsby'
-import logo from '../img/vocal-arts-logo-full-color.svg'
```

Comment:
The difference between pulling images out of `/static...` and not is if the images will always be there or not.
If they are being pulled from `/static/` that means the admin of the system can edit the image and take it away. Since we're import it directly here (not going through gatsby's system of graphql calls) we should make sure the image is always there. So we should leave direct imports like this or change it so the admin can edit the photo.